wormhole-connect/src/routes/ntt/chains/evm/getMessage.ts

```diff
@@ -12,7 +12,8 @@ import { getTokenById } from 'utils';
 import { getWormholeLogEvm } from 'utils/vaa';
 import config from 'config';
 import { toChainName } from 'utils/sdk';
-import { deserializePayload, Ntt } from '@wormhole-foundation/sdk-definitions';
+import { deserializePayload } from '@wormhole-foundation/sdk-definitions';
+import { Ntt } from '@wormhole-foundation/sdk-definitions-ntt';
```

Comment:
Needed to bump the `sdk-definitions` package and it no longer contains the NTT namespace, so now we're using `sdk-definitions-ntt` which is what should be used going forward.